src/metatrain/experimental/phace/default-hypers.yaml

```diff
@@ -0,0 +1,43 @@
+architecture:
+  name: experimental.phace 
+  
+  model:
+    nu_max: 3
```

Comment:
should this be called `body_order_max`? Might be more directly understandable by users